src/System/GameDataFunction.h

```diff
@@ -0,0 +1,499 @@
+#pragma once
+
+#include <basis/seadTypes.h>
+#include <math/seadVectorFwd.h>
+#include <prim/seadSafeString.h>
+#include <nn/g3d/ResFile.h>
+#include <math/seadMatrix.h>
+
+class GameDataHolderWriter;
+class ChangeStageInfo;
+class GameDataHolderAccessor;
+class ShineInfo;
+class GameDataHolder;
+class GameDataFile;
+class RiseMapPartsHolder;
+class Shine;
+class QuestInfo;
+class SessionEventProgress;
+class WorldTravelingNpc;
+class BindInfo;
+class ScenarioStartCamera;
+class CollectBgm;
+struct HackObjInfo;
+struct ShopItemInfo;
+
+namespace al {
+    class LiveActor;
+    class PlacementId;
+    class IUseSceneObjHolder;
+    class SceneObjHolder;
+    class GameDataHolderBase;
+    class LayoutActor;
+    class ActorInitInfo;
+    class PlacementInfo;
+    class Scene;
+    class IUseMessageSystem;
+    class CameraTicket;
+} // namespace al
+
+namespace GameDataFunction {
```

Comment:
namespace or class?

Reply:
doesn't really matter, as long as all functions are made static. namespace looks cleaner IMO, since there's no need see "static" on every single function, but afaik there's no way to know definitively so i chose namespace.

Reply:
According to recent discussions, should probably be `class` instead